SQL/0000-00-00-schema.sql

```diff
@@ -368,6 +370,24 @@ CREATE TABLE `test_names` (
   CONSTRAINT `FK_test_names_1` FOREIGN KEY (`Sub_group`) REFERENCES `test_subgroups` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `test_json` (
+  `TestJsonID` varchar(64) NOT NULL,
+  `JsonData` TEXT NOT NULL,
+  CONSTRAINT `PK_test_json` PRIMARY KEY (`TestJsonID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `test_names_json_rel` (
+  `TestNamesID` int(10) unsigned NOT NULL,
+  `TestJsonID` varchar(64) NOT NULL,
+  `DateUpdated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `UsersID` int(10) unsigned NOT NULL,
+  `Current` BIT NOT NULL,
```

Comment:
just to make it clearer maybe `currentlyActive` is more dsciptive

Reply:
yes sir

Reply:
@ridz1208 what about just `Active`?